examples/with-styled-components-and-scoping/README.md

```diff
@@ -0,0 +1,48 @@
+# Docz Styled Components Example
+
+## Using `create-docz-app`
+
+```sh
+npx create-docz-app docz-app-styled-docz --example styled-components
```

Comment:
styled-components -> with-styled-components-and-scoping

Reply:
I hope i found them all now ^^